tap_harvest/__init__.py

```diff
@@ -306,11 +308,11 @@ def map_invoice_payment(payment):
 
 
 def sync_estimates():
-    def map_estimate_message(message):
-        message['estimate_id'] = message['id']
-        return message
-
     def for_each_estimate(estimate, time_extracted):
+        def map_estimate_message(message):
```

Comment:
Why we changed this code?

Reply:
Because in the records mapping was not right, `estimate_id` and `id` of each estimate message was the same

Reply:
Can you put a small comment over it?